### ISSUE TITLE
model.finetune(...) does not show the loss of the model

### ISSUE THREAD

**Issue by JessicaLopezEspejel** [google-research/text-to-text-transfer-transformer#1045]
Hello, 

I just run the following jupyter notebook: [t5-trivia.ipynb](https://github.com/google-research/text-to-text-transfer-transformer/blob/main/notebooks/t5-trivia.ipynb) and it works well, however, when I run `model.finetune(...)` does not show the loss of the model.

Any idea how can I solved it? 

Thank you

**Comment by antonio-mastropaolo** [comment]
Has anyone managed to solve this annoying problem?

**Comment by JessicaLopezEspejel** [comment]
Hello @antonio-mastropaolo, 

Please add this line in your notebook: ```from tqdm.notebook import tqdm```

For me it worked, let me know if it works for you.

**Comment by antonio-mastropaolo** [comment]
Hello @JessicaLopezEspejel 

Thanks a lot for getting back to me.
Unfortunately I didn't solve it yet.

My code looks like this:

```
from tqdm.notebook import tqdm
TRAIN_STEPS =  100000#@param {type: "integer"}

with gin.unlock_config():
    gin.parse_config_file(LOCAL_GIN_PATH)
    model.finetune(mixture_or_task_name=MIXTURE_NAME,
               finetune_steps=TRAIN_STEPS,
               pretrained_model_dir=PRETRAIN_MODEL_DIR)

```

**Comment by adarob** [comment]
Can you provide more details? Is the model training? Are results showing up in the tensorboard in the previous cell?

**Comment by antonio-mastropaolo** [comment]
Hello @adarob 

Thanks for chiming in

Yep, we are talking about the model training.
As for the tensorboard, I'm not using it

**Comment by coreyfournier** [comment]
I'm pretty sure it outputs it to graph.pbtxt and Tensorboard reads it and graphs the loss.
